client/boot/common.js

```diff
@@ -410,6 +411,11 @@ const setupMiddlewares = ( currentUser, reduxStore ) => {
 			featureHelper( document.querySelector( '.environment.is-features' ) );
 		} );
 	}
+
+	if ( config.isEnabled( 'logmein' ) && currentUser && currentUser.get() ) {
```

Comment:
The logged-in check should be reading from Redux: `isUserLoggedIn( reduxStore.getState() )`. @tyxla is working on removing the `lib/user` helper object.

Reply:
Thanks for the heads up